atomix/cluster/src/main/java/io/atomix/raft/protocol/PersistedRaftRecord.java

```diff
@@ -1,19 +1,11 @@
 /*
- * Copyright © 2020 camunda services GmbH (info@camunda.com)
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
  */
-package io.atomix.raft.storage.log;
+package io.atomix.raft.protocol;
```

Comment:
:question: Why was this moved?

Reply:
Even if it has `Persisted` in the name, this class is a DTO - it's used to transmit `IndexedRaftLogEntry` over the network during replication, then used to create a new one on the receiving side. It's never really used to interact with log itself except in the context of network replication.